crates/relayer-cli/src/commands/query/packet/commitment.rs

```diff
@@ -82,7 +82,7 @@ impl QueryPacketCommitmentCmd {
                     sequence: self.sequence,
                     height: self.height.map_or(QueryHeight::Latest, |revision_height| {
                         QueryHeight::Specific(
-                            ibc::Height::new(chain.id().version(), revision_height)
+                            ibc_relayer_types::Height::new(chain.id().version(), revision_height)
```

Comment:
Is the `ibc_relayer_types::` required since it is imported at line 10 ? A similar case in file `crates/relayer-cli/src/commands/query/packet/ack.rs` should we have `use ibc_relayer_types::Height` and call directly `Height` or should we remove `use ibc_relayer_types::Height` and use directly `ibc_relayer_types::Height` instead ? Not sure which is preferred, but I think being consistent everywhere would be nice.

Reply:
Agreed let's use the non-qualified version, ie. just Height